### ISSUE TITLE
Chapter Editor pages skip several controls in forwards navigation by keyboard 

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1648]
### Description

Focus order for the chapter, front matter, back matter, and part pages does not follow a logical sequence, making these pages harder to navigate by keyboard. 

Example: On the Edit Chapter page, focus moves from the chapter title text input to the text entry area, skipping the controls and links in between. Some controls, such as the permalink, Add Media button and Visual/Text tabs can be reached, but only when navigating backwards. As a result, non-visual users may not be aware that these buttons exist. 

### Steps to Reproduce

1. Navigate through any chapter editor page by tabbing from the top of the page down 
2. Note deviations from the logical sequence of settings 

![Screen Shot 2019-04-04 at 2 18 03 PM](https://user-images.githubusercontent.com/39064028/55578680-78291000-56e4-11e9-9f42-79f30b76b2f1.png)

**Comment by SteelWagstaff** [comment]
Focus order is controlled upstream by WordPress and cannot be easily modified by us.

Users will see keyboard shortcut option `shift + option + H` when screenreader reads page contents. In that menu, they'll see the `Alt + F10` command to shift focus to the editor buttons keys.
